test/test_torch.py

```diff
@@ -6614,9 +6614,9 @@ def test_parsing_intlist(self):
 
         # fail parse with > 1 element variables
         self.assertRaises(TypeError, lambda: torch.ones(torch.tensor(3, 3)))
-        self.assertRaises(TypeError, lambda: torch.ones((torch.tensor(3, 3))))
+        self.assertRaises(TypeError, lambda: torch.ones(torch.tensor(3, 3)))
+        self.assertRaises(TypeError, lambda: torch.ones(np.array(3, 3)))
         self.assertRaises(TypeError, lambda: torch.ones(np.array(3, 3)))
-        self.assertRaises(TypeError, lambda: torch.ones((np.array(3, 3))))
```

Comment:
This one seems intentionally a tuple, so it would be good to ensure this is not breaking intentionality.

Also, if this is a safe change, lines 6618 and 6619 look the same now, so one can be removed.

Reply:
`(np.array(3, 3))` is not a tuple. It needs a trailing comma to be a tuple (`(np.array(3, 3),)`). So maybe it's a bug from the test?

Reply:
So maybe we should fix these bugs? After having the confirmation that they are actual bugs. Because it'd be weird to have this duplicate line otherwise

Reply:
Okay, my interpretation is that these intended to be tuples, so I fixed them now.